pandas/tests/series/test_constructors.py

```diff
@@ -970,16 +970,34 @@ def test_constructor_with_naive_string_and_datetimetz_dtype(self, arg):
         expected = Series(pd.Timestamp(arg)).dt.tz_localize("CET")
         assert_series_equal(result, expected)
 
-    def test_construction_interval(self):
+    @pytest.mark.parametrize("interval_constructor", [IntervalIndex, IntervalArray])
+    def test_construction_interval(self, interval_constructor):
```

Comment:
no material change here, just cleaning up by using  `parametrize` and removing some unnecessary lines (unassigned `repr`/`str` calls)